packages/screens/Mini/components/MiniCommentInput/CommentTextInput.tsx

```diff
@@ -95,7 +98,7 @@ export const CommentTextInput = React.forwardRef<MiniCommentInputHandle, Props>(
                 }
                 onBlur={onBlur}
                 onFocus={onFocus}
-                placeholder="Hey yo! Write your comment"
+                placeholder={`Hey yo! ${type === "comment" ? "Write your comment" : "Post something"}`}
```

Comment:
I know is not related to this PR.
I think we should change the "Hey yo" for something else.